projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/graphql/strategy/datajob/JobFieldStrategyBy.java

```diff
@@ -29,7 +29,8 @@
    DESCRIPTION("config.description", "content/config/description"),
    SOURCE_URL("config.sourceUrl", "content/config/sourceUrl"),
    NEXT_RUN_EPOCH_SECS("config.schedule.nextRunEpochSeconds", "content/config/schedule/nextRunEpochSeconds"),
-   SCHEDULE_CRON("config.schedule.scheduleCron", "content/config/schedule/scheduleCron");
+   SCHEDULE_CRON("config.schedule.scheduleCron", "content/config/schedule/scheduleCron"),
+   DATA_JOB_CONTACTS("config.contacts.present", "content/config/contacts/present");
```

Comment:
what is present?

Reply:
I am askng because our GraphQL model should be similar to our API model which is resource based. 
So when you query graphql you get a list of resources (data jobs in our case) which some fields being filtered out - and also possibly sorted by some fields. 

Each data job can have configuration which can have contracts. 

https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/projects/pipelines_control_service/src/main/resources/schema.graphqls#L119

Reply:
If we have a DataJobContact present or not. For more context see this conversation https://github.com/vmware/versatile-data-kit/pull/582#pullrequestreview-819074713 . The config.contacts.present field is intended to be used only when sorting. This type of sort was requested by users.

Reply:
We don't have to since we aren't planning on returning anything associated with this field and if we update it, if this field is requested the returned value will always be null.

Reply:
Actually, @tozka has a point. We should consider adding this field to the schema, but definitely in separate PR.

Reply:
The problem with adding it to the schema is that it will require a large (ish) refactor since right now DataJobContact is a class generated by the openApi specification that we return as is. This means that we will have to have another V2DataJobContact class to return in the graphQL API which will have the extra property and translate the openApi generated class? Or is there another solution?

Reply:
We just need to add the new column to the DataJobContact.

Reply:
Created https://github.com/vmware/versatile-data-kit/issues/610